File: python/nested if_else.py
x = int(input("enter the number1 :"))
y = int(input("enter the number2 :"))
z = int(input("enter the number3 :"))
if x > y :
    if x > z :
        print(x ,"is the greatest number")
    else :
        print(z , "is the greatest number")
else :
    if y > z :
        print(z , "is the greatest number")
    else :
        print(z ,"is the greatest number")

File: CP in c++/c/nestedIfElse.c
#include<stdio.h>
int main(){
    int x;
    printf("enter the number1: ");
    scanf("%d" , &x);
    int y;
    printf("enter the number2: ");
    scanf("%d" , &y);
    int z;
    printf("enter the number3: ");
    scanf("%d" , &z);
    if (x >= y){
        if (x >= z) printf("the greatest number is: %d" , x);
        else printf("the greatest number is: %d" , z);
    }
    else {
        if (y >= z) printf("the greatest number is: %d" , y);
        else printf("the greatest number is: %d" , z);
    }
    return 0 ;
}